main.py

```diff
@@ -248,10 +273,13 @@ def data():
                    battery_temperature=randint(80, 120),
                    bms_fault=choices([0, 1], weights=[.9, .1])[0],
                    gps_time=int(time()),
-                   timestamp=int(time()), # seconds since epoch
+				   timestamp=int(time()), # seconds since epoch
                    gps_lat=None,
                    gps_lon=None,
-                   gps_speed=randint(30, 40),
+                   gps_velocity_east=None,
```

Comment:
We aren't actually using these gps_velocity_* fields, right? Can we just remove them?